interfaces/builtin/audio_playback.go

```diff
@@ -84,6 +84,10 @@ const audioPlaybackConnectedPlugAppArmorCore = `
 owner /run/user/[0-9]*/###SLOT_SECURITY_TAGS###/pulse/ r,
 owner /run/user/[0-9]*/###SLOT_SECURITY_TAGS###/pulse/native rwk,
 owner /run/user/[0-9]*/###SLOT_SECURITY_TAGS###/pulse/pid r,
+# To allow to use pipewire-pulse in system mode, instead of user mode
+owner /var/snap/###SLOT_SECURITY_TAGS_NO_SNAP###/common/pulse/ r,
+owner /var/snap/###SLOT_SECURITY_TAGS_NO_SNAP###/common/pulse/native rwk,
+owner /var/snap/###SLOT_SECURITY_TAGS_NO_SNAP###/common/pulse/pid r,
```

Comment:
Should this be called `SLOT_INSTANCE_NAME` to avoid the complex "this is a security tag but it's not a security tag" semantics of the name?

Reply:
Yeah, completely agree. Fixing it.